docs/docs/widgets/pagination.md

```diff
@@ -0,0 +1,71 @@
+---
+id: pagination
+title: Pagination
+---
+# Pagination
+
+Pagination enables the user to select a specific page from a range of pages.
+
+:::tip
+It is used seperate content into discrete pages.
+:::
```

Comment:
`It is used seperate content into discrete pages` can be moved in the description and inside tip maybe we can add something like `You can club Pagination widget with the List View widget.` Or any other use case if you have in your mind.